src/App.js

```diff
@@ -98,7 +104,12 @@ class App extends Component {
       reportsCompArr: [],
       minReportsArr: [],
       hiddenReports: {},
-      metric: null
+      metric: null,
```

Comment:
is it being used?

Reply:
For another feature! only styling change here